include/albatross/src/models/ransac.hpp

```diff
@@ -91,29 +91,52 @@ inline bool ransac_success(const ransac_return_code_t &rc) {
   return rc == RANSAC_RETURN_CODE_SUCCESS;
 }
 
+template <typename GroupKey> struct RansacIteration {
+  std::vector<GroupKey> candidates;
+  std::map<GroupKey, double> inliers;
+  std::map<GroupKey, double> outliers;
+  double consensus_metric;
```

Comment:
Would "consensus_score" or "consensus_metric_score" be more intuitive, so that it doesn't sound like a function.

Reply:
I changed it to `consensus_metric_value`.